pyproject.toml

```diff
@@ -1,3 +1,30 @@
+# pyproject.toml
+[build-system]
+requires      = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "python_open_feature_sdk"
```

Comment:
We may want to change this - I'm not sure if there are any conventions we need to follow. But it's not important for this PR.

Reply:
Easy enough to change at another point

Reply:
Agreed.